File: c++/kode program/Produk.cpp
#include "Produk.h"
#include <iomanip>

Produk::Produk(int id, std::string Nama_produk, float Harga, int Stok, std::string GambarPath)
    : id(id), Nama_produk(Nama_produk), Harga(Harga), Stok(Stok), GambarPath(GambarPath) {}

void Produk::display() const {
    std::cout << std::left
              << std::setw(5) << id
              << std::setw(20) << Nama_produk
              << std::setw(10) << static_cast<int>(Harga)
              << std::setw(5) << Stok
              << std::setw(15) << GambarPath;
}

int Produk::getid() const {
    return id;
}

File: python/kode program/Produk.py
class Produk:
    def __init__(self, id, Nama_produk, Harga, Stok, GambarPath):
        self.id = id
        self.Nama_produk = Nama_produk
        self.Harga = Harga
        self.Stok = Stok
        self.GambarPath = GambarPath

    def display(self):
        print(f"{self.id:<5} {self.Nama_produk:<20} {self.Harga:<10} {self.Stok:<5} {self.GambarPath:<15}", end=" ")

    def getid(self):
        return self.id
